package.json

```diff
@@ -276,28 +261,19 @@
         "command": "PowerShell.ClosePanel",
         "title": "Close panel",
         "category": "PowerShell",
-        "icon": {
-          "light": "media/resources/light/ClosePanel.svg",
-          "dark": "media/resources/dark/ClosePanel.svg"
-        }
+        "icon": "$(layout-activitybar-left)"
```

Comment:
This and the next two are new and in 1.65.0. They're only seen in the ISE-mode. We could delay changing them until then. I guess Azure Data Studio users that use ISE-mode wouldn't see these, it'd look like this:
<img width="196" alt="Screen Shot 2022-03-04 at 2 40 05 PM" src="https://user-images.githubusercontent.com/2226434/156851644-141f94ee-2e9d-4cac-928e-44f5adb77aeb.png">

Ok ok, we'll stash this part of the change for later.